nml/actions/action2layout.py

```diff
@@ -208,7 +208,8 @@ def get_all_registers(self):
 
     def create_register(self, name, value):
         if (
-            isinstance(value, expression.StorageOp)
+            self.feature != 0x04
```

Comment:
Maybe a comment would be helpful here on why stations are excluded.

Reply:
Like 
```suggestion
            self.feature != 0x04 # Stations may use STORE_TEMP() after the layout registers init.
```
?